src/Elastic.Apm/Model/Payload/Stacktrace.cs

```diff
@@ -1,9 +1,11 @@
-﻿using Newtonsoft.Json;
+﻿using Elastic.Apm.Report.Serialization;
+using Newtonsoft.Json;
 
 namespace Elastic.Apm.Model.Payload
 {
 	internal class Stacktrace
 	{
+		[NoTruncationInJsonNet]
```

Comment:
Would it be cleaner to declare the reverse? `[Truncate(int chars)]` instead?

Reply:
I just spotted @skynet2 comment. I tend to agree, a default truncate feels implicit rather than explicit.

Reply:
Ok, since my logic seems to be strange for basically everyone, we go with the reverse. 

Just for reference: the main reason for trimming by default was this:
If we trim by default and we forget to add `[DontTruncate]` -> We trim the string, but the first 1024 chars and everything else is visible.
If we don't trim and forget to add `[Truncate]` -> The server rejects the request (the whole transaction or span), nothing is visible. 

Of course we should not forget to add attributes in the first place :) 

Anyway, I accept that trimming by default would be strange. Reversed the logic.